compiler/src/dotty/tools/dotc/core/SymbolLoaders.scala

```diff
@@ -19,15 +19,12 @@ import parsing.Parsers.OutlineParser
 import reporting.trace
 
 object SymbolLoaders {
+  import ast.untpd._
+
   /** A marker trait for a completer that replaces the original
    *  Symbol loader for an unpickled root.
    */
   trait SecondCompleter
-}
-
-/** A base class for Symbol loaders with some overridable behavior  */
```

Comment:
It seems that `SymbolLoaders` was meant to be extended. Make the `protected` members `private`?